Library/Homebrew/utils/github.rb

```diff
@@ -285,14 +266,42 @@ def print_pull_requests_matching(query)
       puts "Closed pull requests:"
       prs = open_or_closed_prs
     else
-      return
+      return []
     end
 
     prs.each { |i| puts "#{i["title"]} (#{i["html_url"]})" }
   end
 
   def private_repo?(full_name)
-    uri = URI.parse("#{API_URL}/repos/#{full_name}")
+    uri = url_to "repos", full_name
     open(uri) { |json| json["private"] }
   end
+
+  def query_string(*main_params, **qualifiers)
+    params_list = main_params
+
+    qualifiers.each do |key, value|
+      if value.is_a? Array
```

Comment:
Instead of these separate checks, just use `[*value]`, and loop through all values, even if it is only one.